pkg/asset/machines/master.go

```diff
@@ -169,6 +172,11 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 				}
 			}
 		}
+		mpool.InstanceType, err = aws.PreferedInstanceType(ctx, installConfig.AWS, awsDefaultMasterMachineTypes(installConfig.Config.Platform.AWS.Region), mpool.Zones)
```

Comment:
Need to make sure this defaults are only done when no instance type set by user.

Reply:
fixed with https://github.com/openshift/installer/pull/3051/commits/de04fffd44630d368d461c555613286557168792